DependencyInjection/Configuration.php

```diff
@@ -225,4 +227,25 @@ private function addContentListenerSection(NodeBuilder $nodeBuilder)
             ->end()
         ;
     }
+
+    /**
+     * Attach the form node to the tree.
+     *
+     * @param NodeBuilder $nodeBuilder
+     */
+    private function addFormSection($nodeBuilder)
+    {
+        $nodeBuilder
+           ->arrayNode('form')
+                ->canBeDisabled()
```

Comment:
had some complications with the existence of the the embedded array and so on, will try to do that better.